eZ/Publish/Core/settings/fieldtype_services.yml

```diff
@@ -120,3 +120,17 @@ services:
     eZ\Publish\Core\FieldType\RichText\CustomTagsValidator:
         public: false
         arguments: ['%ezplatform.ezrichtext.custom_tags%']
+
+    eZ\Publish\Core\FieldType\ImageAsset\NameableField:
+        public: false
+        arguments:
+            $handler: '@ezpublish.spi.persistence.cache.contentHandler'
+        tags:
+            - {name: ezpublish.fieldType.nameable, alias: ezimageasset}
+
+    eZ\Publish\Core\FieldType\ImageAsset\AssetMapper:
+        arguments:
+            $contentService: '@ezpublish.api.service.content'
+            $locationService: '@ezpublish.api.service.location'
+            $contentTypeService: '@ezpublish.api.service.content_type'
+            $mappings: '$fieldtypes.ezimageasset.mappings$'
```

Comment:
Since the setting is introduced on Bundle level, here you should rather have some core fallback. It will also allow you to avoid overriding service in test yamls.